frontend/src/scenes/retention/retentionModalLogic.ts

```diff
@@ -46,30 +46,30 @@ export const retentionModalLogic = kea<retentionModalLogicType>([
                 return aggregationLabel(aggregation_group_type_index)
             },
         ],
-        personsQuery: [
+        ActorsQuery: [
```

Comment:
Should this be PascalCase? We usually camelCase variables and PascalCase types and classes 🤷

Reply:
Hmm, good catch.. I thought I did case sensitive search and replace 👀